modules/virt-hot-plugging-bridge-network-interface-cli.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-hot-plugging-network-interfaces.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-hot-plugging-bridge-network-interface_{context}"]                                   
+= Hot plugging a bridge network interface using the CLI
+
+Hot plug a bridge network interface to a virtual machine (VM) while the VM is running. 
+
+.Prerequisites
+
+* The Multus Container Network Interface (CNI) plugin is deployed on the cluster.
```

Comment:
I just clarified this with Petr Horacek. D/S, no need to mention CNI as prerequisite because it is installed with CNV.